src/notebook/output-area/model.ts

```diff
@@ -167,14 +167,14 @@ class OutputAreaModel implements IDisposable {
       let future = kernel.execute(content);
       // Handle published messages.
       future.onIOPub = (msg: KernelMessage.IIOPubMessage) => {
-        let msgType = msg.header.msg_type as nbformat.OutputType;
+        let msgType = msg.header.msg_type;
         switch (msgType) {
         case 'execute_result':
         case 'display_data':
         case 'stream':
         case 'error':
           let model = msg.content as nbformat.IOutput;
-          model.output_type = msgType;
+          model.output_type = msgType as nbformat.OutputType;
```

Comment:
There is a similar line in console/content.ts that could also be updated. I'll put a PR on your PR if you don't want to go digging.